.gitignore

```diff
@@ -36,6 +36,7 @@ node_modules
 lib
 esm
 .idea
+*.iml
```

Comment:
i'm not sure what this is, but generally things like this (non-project-specific files) go in your global gitignore file rather than every project you touch.